test/library.cpp

```diff
@@ -347,14 +352,33 @@ class LibraryTest : public ::testing::Test {
   std::shared_ptr<kiwix::Library> lib;
 };
 
+TEST_F(LibraryTest, createBookMark)
+{
+  auto bookId = "0c45160e-f917-760a-9159-dfe3c53cdcdd";
+  auto book = lib->getBookById(bookId);
+
+  auto bookmark = createBookmark(book, "/a/url", "A title");
+
+  EXPECT_EQ(bookmark.getUrl(), "/a/url");
+  EXPECT_EQ(bookmark.getTitle(), "A title");
+  EXPECT_EQ(bookmark.getBookId(), bookId);
+  EXPECT_EQ(bookmark.getBookName(), book.getName());
+  EXPECT_EQ(bookmark.getBookName(), "wikipedia_fr_tunisie");
+  EXPECT_EQ(bookmark.getBookFlavour(), book.getFlavour());
+  EXPECT_EQ(bookmark.getLanguage(), book.getCommaSeparatedLanguages());
```

Comment:
What is the reason that `Bookmark::getBookName()` is tested against both return value of `Book::getName()` and an explicit string value, whereas `getBookFlavour()` and `getLanguage()` receive a different treatment?